sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_table_storage_timeline_daily_v1/query.py

```diff
@@ -60,7 +65,9 @@ def main():
     args = parser.parse_args()
 
     partition = args.date.replace("-", "")
-    destination_table = f"{args.project}.{args.destination_dataset}.{args.destination_table}${partition}"
+    destination_table = f"""
```

Comment:
Should double check this, but I think this will create the string with `\n` (newline character) at the beginning and the end of the string which might result in weird behaviour.